Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -292,6 +292,70 @@ void CFrame::OpenGeneralConfiguration(wxWindowID tab_id)
   m_main_config_dialog->SetFocus();
 }
 
+void CFrame::CycleProfile(bool forward) {
+  //load the hotkey config
```

Comment:
Comments should have a space after the initial forward slashes (`// Load`)

Reply:
Fixed.